data/json/monsters.json

```diff
@@ -5963,6 +5963,37 @@
     "flags":["SEES", "HEARS", "SMELLS", "WARM", "BASHES", "GROUP_BASH", "POISON", "BLEED", "CBM_OP", "ACIDPROOF", "NO_BREATHE", "REVIVES", "BONES"]
     },{
     "type" : "MONSTER",
+    "id" : "mon_zombie_armored",
+    "name": "decayed paladin",
+    "species":"ZOMBIE",
+    "default_faction":"zombie",
+    "symbol":"Z",
+    "color":"dkgray_red",
+    "size":"MEDIUM",
+    "material":"flesh",
+    "diff":25,
+    "aggression":100,
+    "morale":100,
+    "speed":70,
+    "melee_skill":5,
+    "melee_dice":2,
+    "melee_dice_sides":6,
+    "melee_cut":0,
+    "dodge":0,
+    "armor_bash":24,
+    "armor_cut":24,
+    "vision_day":30,
+    "vision_night":3,
+    "item_chance":80,
+    "death_drops": "mon_zombie_armored_death_drops",
+    "luminance":0,
+    "hp":120,
+    "death_function":["NORMAL"], 
+    "description":"Despite being encased in a nigh impenetrable suit of power armor, something still managed killed this soldier.",
```

Comment:
For a power armor to become 24/24, it would have to be totally wrecked. Player salvaged armor can only go as low as 36 before it gets destroyed.
Maybe drop the power armored thing and just make it a special survivor zed who managed to find a plate/chain suit? It would fit the stats. Plate armors and chainmails get no love.

If the power armor thing is to stay, maybe add something like HARDTOHIT, except with attacks failing rather than missing? It would make sense that all attacks glance off the intact parts, but face only minor resistance (24 is minor here) on the smashed apart ones.